config/settings/base.py

```diff
@@ -239,7 +239,7 @@
     "CELERY_BROKER_CONNECTION_RETRY_ON_STARTUP", default=True
 )
 # http://docs.celeryproject.org/en/latest/userguide/configuration.html#std:setting-result_backend
```

Comment:
Could you update the doc link? 
https://docs.celeryq.dev/en/latest/userguide/configuration.html#database-backend-settings